pkg/types/imagebased/imagebased_config_types.go

```diff
@@ -0,0 +1,109 @@
+package imagebased
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	aiv1beta1 "github.com/openshift/assisted-service/api/v1beta1"
+	"github.com/openshift/installer/pkg/types"
+)
+
+// ImageBasedConfigVersion is the version supported by this package.
+const ImageBasedConfigVersion = "v1beta1"
+
+// InstallationConfig is the API for specifying configuration for the
+// image-based installation ISO.
+type InstallationConfig struct {
```

Comment:
I wonder if we can use lca struct here? or should we import this one in lca?

Reply:
That's a good question. I initially thought that we should bring everything here, in order to have a clear idea on what this config, that's going to be used with the `openshift-install imagebased` command, contains and import this in LCA. But I'm still trying to justify this choice further. 

Do you see any big wins for either of those 2 choices (keep the struct here or in LCA)?

Reply:
We will need to update both for each change rather than just import it

Reply:
Yes, any change to this struct, wherever it is, will most probably lead to changes in both the installer and LCA anyway. 

I think one upside of keeping it here is that we don't need to import/vendor the `release-*` branch of the respective LCA pkg, which we might end up doing for every OpenShift release (as LCA ships alongside OpenShift). 

Other than that I don't have any objections of importing/vendoring the LCA struct here, but we would first need to adjust it to reflect all the changes we're proposing in this PR, i.e. remove the [UseContainersFolder](https://github.com/openshift-kni/lifecycle-agent/blob/main/api/ibiconfig/ibiconfig.go#L97), the [RHCOSLiveISO](https://github.com/openshift-kni/lifecycle-agent/blob/main/api/ibiconfig/ibiconfig.go#L27), the [Precache*](https://github.com/openshift-kni/lifecycle-agent/blob/main/api/ibiconfig/ibiconfig.go#L82-L88) and perhaps the [ExtraPartition*](https://github.com/openshift-kni/lifecycle-agent/blob/main/api/ibiconfig/ibiconfig.go#L99-L115) fields. 

WDYT? Should we keep it as is here or adjust the LCA one and import/vendor it?

Reply:
Works for me but we need to have some comment maybe in lca object that changes in it requires change here?

Reply:
That's probably a good idea, I'll leave this discussion unresolved to make sure that we add such a comment when this is merged.